### PR TITLE
Ins 2996 has pendings

### PR DESCRIPTION
**Pull request by aMFeriuSS**
**- What I did**
New handler and procedure for watermill HasPendings message

### CODE REVIEW

File: insolar/payload/payload.proto [hunk 1]
```diff
@@ -207,3 +207,15 @@ message GetPendings {
...
     bytes ObjectID = 20 [(gogoproto.customtype) = "github.com/insolar/insolar/insolar.ID", (gogoproto.nullable) = false];
 }
+
+message HasPendings {
+    uint32 Polymorph = 16;
+
+    bytes ObjectID = 20 [(gogoproto.customtype) = "github.com/insolar/insolar/insolar.ID", (gogoproto.nullable) = false];
+}
+
+message Has {
```
Comment (andreyromancev) [inline insolar/payload/payload.proto]: Let's rename it. Something more specific like `PendingInfo`.

File: insolar/payload/payload.go [hunk 2]
```diff
@@ -367,6 +378,26 @@ func Unmarshal(data []byte) (Payload, error) {
...
+		return &pl, err
+	case TypePendingsInfo:
+		pl := PendingsInfo{}
+		err := pl.Unmarshal(data)
+		return &pl, err
+	case TypeReplication:
+		pl := Replication{}
+		err := pl.Unmarshal(data)
+		return &pl, err
+	case TypeResultInfo:
```
[the remark this answers is not in the document]
Comment (aMFeriuSS) [reply insolar/payload/payload.go]: done

File: ledger/light/proc/has_pendings.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	}
+
+	msg, err := payload.NewMessage(&payload.PendingsInfo{
+		HasPendings: idx.Lifeline.EarliestOpenRequest != nil && *idx.Lifeline.EarliestOpenRequest < flow.Pulse(ctx),
+	})
+	if err != nil {
+		return errors.Wrap(err, "failed to create reply")
+	}
+
+	go hp.dep.sender.Reply(ctx, hp.message, msg)
```
Comment (andreyromancev) [inline ledger/light/proc/has_pendings.go]: Le'ts remove `go`.

Comment (aMFeriuSS) [reply ledger/light/proc/has_pendings.go]: done

**Pull request merged**